app/src/renderer/components/staking/LiValidator.vue

```diff
@@ -1,28 +1,18 @@
 <template lang='pug'>
-li.li-validator(:class='styles')
-  router-link(:to="{ name: 'validator', params: { validator: validator.id }}")
-    .li-validator__value.name
+tr.li-validator
```

Comment:
probably need to change the name of this component at some point since it's no longer an `<li>`.